src/views/LoginView/Login.jsx

```diff
@@ -1,24 +1,24 @@
-import React, {Component} from 'react';
-import { connect } from 'react-redux';
-import PropTypes from 'prop-types';
-import LoginView from '../../components/Login/LoginForm';
-import login from '../../actions/LoginAction';
- 
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import LoginView from "../../components/Login/LoginForm";
+import login from "../../actions/LoginAction";
+
 /**
  * A class that holds all the logic for login
  */
-export class Login extends Component{
+export class Login extends Component {
   /**
-   * 
+   *
    * @param {} props - initial props of login class
    */
-  constructor(props){
-    super(props)
+  constructor(props) {
+    super(props);
     this.state = {
       email: "",
       password: "",
       errors: ""
-    }
+    };
     this.handleChange = this.handleChange.bind(this);
```

Comment:
Use arrow function syntax so that you don't need to bind them in the constructor.